openbas-front/src/components/common/ButtonPopover.tsx

```diff
@@ -39,11 +44,25 @@ const ButtonPopover: FunctionComponent<Props> = ({
         onClose={() => setAnchorEl(null)}
       >
         {entries.map((entry, idx) => {
+          if (entry.type === 'export') {
+            return (
+              <MenuItem key={idx}>
+                <CSVLink
+                  data={entry.data || []}
+                  filename={entry.filename || 'data.xls'}
+                >
+                  {t(entry.label)}
+                </CSVLink>
```

Comment:
It's not supposed to be here
This component is generic and you are putting a lot of specific stuff inside.